mtools/util/logevent.py

```diff
@@ -653,9 +664,19 @@ def _extract_counters(self):
                         try:
                             # Remap counter to standard name, if applicable
                             counter = counter_equiv.get(counter, counter)
-                            vars(self)['_' + counter] = int((token.split(':')
-                                                             [-1]).replace(',',
-                                                                           ''))
+                            #extract allowDiskUse counter
+                            if(counter == 'allowDiskUse' and
+                                token.startswith('allowDiskUse')):
+                                try:
+                                    self._allowDiskUse = (
+                                        split_tokens[t + 1 + self.datetime_nextpos + 2].replace(',', ''))
```

Comment:
ok what exactly is being calculated here?

You might comment this better because that calculation is not clear, what in the log line are you targeting exactly?

also no whitespace inside the [] :)